src/pages/auth/chat/list/ChatList.tsx

```diff
@@ -26,7 +58,7 @@ export default function ChatList(props: propsType) {
         <S.ChatItem head>
           <ChatItem no={"No"} subject={"방제"} owner={"방장"} participantsCnt={"인원"} head />
         </S.ChatItem>
-        {props.chat.map((room) => {
+        {chatList.map((room) => {
           return (
             <S.ChatItem key={no1}>
```

Comment:
```suggestion
            <S.ChatItem key={room.roomId}>
```

key 값은 단순 인덱스보다 고유값으로 사용하는게 좋습니다!